### PR TITLE
Samsung.EasySettingBox version 1.2.1

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/106257)

### CODE REVIEW

**Comment by stephengillie** [comment]
Manifest validation failed:
>  Manifest is invalid. (Line: 7, Col: 17, Idx: 227) - (Line: 7, Col: 17, Idx: 227): Block sequence entries are not allowed in this context.

File: manifests/s/Samsung/EasySettingBox/1.2.1/Samsung.EasySettingBox.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.EasySettingBox
+PackageVersion: 1.2.1
+Installers:
+- InstallerUrl: - https://downloadcenter.samsung.com/content/SW/202301/20230106083833260/EasySettingMSIX64.msi
```
Comment (stephengillie) [inline manifests/s/Samsung/EasySettingBox/1.2.1/Samsung.EasySettingBox.installer.yaml:7]:
```suggestion
- InstallerUrl: https://downloadcenter.samsung.com/content/SW/202301/20230106083833260/EasySettingMSIX64.msi
```

Comment (stephengillie) [reply manifests/s/Samsung/EasySettingBox/1.2.1/Samsung.EasySettingBox.installer.yaml:7]: The esoteric error here [seems to refer](https://blogs.perl.org/users/brian_d_foy/2010/06/block-sequence-entries-are-not-allowed-in-this-context.html) to the data type implied by the en-dash.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installation failed with exit code -1978335226 
> 2023-05-12 16:26:13.090 [CLI ] ShellExecute installer failed: 1603 CustomAction _183B2A37_C02E_4615_A5F5_A511B36654E5 returned actual error code 1157 (note this may not be 100% accurate if translation happened inside sandbox)
> MSI (s) (BC:1C) [16:26:12:408]: Product: Easy Setting Box -- Error 1723. There is a problem with this Windows Installer package. A DLL required for this install to complete could not be run. Contact your support personnel or package vendor.  Action _183B2A37_C02E_4615_A5F5_A511B36654E5, entry: _EsbInstall@4, library: C:\Program Files\Samsung\Easy Setting Box\InstallerHelper.dll  Error 1723. There is a problem with this Windows Installer package. A DLL required for this install to complete could not be run. Contact your support personnel or package vendor.  Action _183B2A37_C02E_4615_A5F5_A511B36654E5, entry: _EsbInstall@4, library: C:\Program Files\Samsung\Easy Setting Box\InstallerHelper.dll  Property(S): ErrorDialog = ErrorDialog
> MSI (s) (BC:1C) [16:26:12:845]: Product: Easy Setting Box -- Installation failed.
> MSI (s) (BC:1C) [16:26:12:845]: Windows Installer installed the product. Product Name: Easy Setting Box. Product Version: 1.2.1. Product Language: 1033. Manufacturer: Samsung. Installation success or error status: 1603.

**Comment by stephengillie** [comment]
This installed normally in a Windows 11 VM:
image

**Comment by upintheairsheep** [comment]
> This installed normally in a Windows 11 VM: image

Why is there an error?

**Comment by upintheairsheep** [comment]
> This installed normally in a Windows 11 VM:
> image

Hardware dependency

**Comment by Trenly** [comment]
> > This installed normally in a Windows 11 VM:
> > image
> 
> Hardware dependency

[Policy] Hardware

**Comment by denelon** [comment]
Supporting custom hardware in validation is "Not Planned".
* https://github.com/microsoft/winget-pkgs/issues/199504

**Pull request closed without merging**